cmd/pinniped/cmd/login_oidc.go

```diff
@@ -179,10 +187,12 @@ func runOIDCLogin(cmd *cobra.Command, deps oidcLoginCommandDeps, flags oidcLogin
 	if flags.credentialCachePath != "" {
 		credCache = execcredcache.New(flags.credentialCachePath)
 		if cred := credCache.Get(cacheKey); cred != nil {
+			pLogger.Debug("using cached cluster credential.")
```

Comment:
```suggestion
			pLogger.Debug("Using cached cluster credential")
```

Small style nit but this message is lowercase compared to others that are sentence-cased. I'd also suggest we standardize on no trailing punctuation (`.`) for consistency.

This also follows the new-ish Kubernetes convention for structured logging: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message


100% not a blocker for merge, just something I noticed.